scripts/us_census/pep/us_pep_sexrace/preprocess.py

```diff
@@ -0,0 +1,172 @@
+    """
+    flag = None
+
+    os.system("mkdir -p " + os.path.join(_MODULE_DIR, 'output_files/final/'))
+    os.system("mkdir -p " +
+              os.path.join(_MODULE_DIR, 'output_files/intermediate/'))
+
+    for config_file in config_files:
+        files = _get_urls(config_file, "urls", test)
+        if "national_1980_1990.json" in config_file:
+            process_national_1980_1990(files)
+        elif "national_1900_1970.json" in config_file:
+            process_national_1900_1970(files)
+        elif "national_1990_2000.json" in config_file:
+            process_national_1990_2000(files)
+        elif "national_2000_2010.json" in config_file:
+            process_national_2000_2010(files)
+        elif "national_2010_2020.json" in config_file:
+            process_national_2010_2020(files)
+        elif "state_1970_1979.json" in config_file:
+            process_state_1970_1979(files)
+        elif "state_1980_1990.json" in config_file:
+            process_state_1980_1990(files)
+        elif "state_1990_2000.json" in config_file:
+            process_state_1990_2000(files)
+        elif "state_2000_2010.json" in config_file:
+            process_state_2000_2010(files)
+        elif "state_2010_2020.json" in config_file:
+            process_state_2010_2020(files)
+        elif "county_1970_1979.json" in config_file:
+            process_county_1970_1979(files)
+        elif "county_1980_1989.json" in config_file:
+            process_county_1980_1989(files)
+        elif "county_1990_2000.json" in config_file:
+            process_county_1990_2000(files)
+        elif "county_2000_2009.json" in config_file:
+            process_county_2000_2009(files)
+        elif "county_2010_2020.json" in config_file:
+            process_county_2010_2020(files)
+
+    # list of national output files before year 2000
+    as_is_output_files = [
+        "nationals_result_1900_1959.csv", "nationals_result_1960_1979.csv",
+        "nationals_result_1980_1990.csv", "nationals_result_1990_2000.csv"
+    ]
+
+    # list of state and county output files before year 2000
+    aggregate_output_files_before = [
+        "state_result_1970_1979.csv", "state_result_1980_1990.csv",
+        "state_result_1990_2000.csv", "county_result_1970_1979.csv",
+        "county_result_1980_1989.csv", "county_result_1990_2000.csv"
+    ]
+
+    # list of state and county output files before after 2000
+    aggregate_output_files_after = [
+        "state_result_2000_2010.csv", "state_result_2010_2020.csv",
+        "county_result_2000_2009.csv", "county_result_2010_2020.csv"
+    ]
+
+    # list of national output files after year 2000
+    geo_aggregate_output_files = [
+        "nationals_result_2000_2010.csv", "nationals_result_2010_2020.csv"
+    ]
+
+    output_files_names = {
+        1: as_is_output_files,
+        2: aggregate_output_files_before,
+        3: aggregate_output_files_after,
+        4: geo_aggregate_output_files
+    }
```

Comment:
Again, maybe we can change the 1,2,3,4 to enums?

Reply:
Used Enum.